src/vsr/replica.zig

```diff
@@ -8028,10 +8061,6 @@ pub fn ReplicaType(
                     self.view > self.view_durable() or
                     self.syncing == .updating_checkpoint,
             );
-            // The primary must only persist the SV headers after repairs are done.
-            // Otherwise headers could be nacked, truncated, then restored after a crash.
```

Comment:
Mhmmmm, I think this comment _still_ holds true, as we never persist any view headers that contain truncated ops. If we persist view headers headers during checkpoint, that ensures our original DVC headers with possibly truncated ops, and we only persist view headers during repair when we have completely repaired our journal from op_repair_min → head op.

It's the assert which doesn't hold true anymore, since we're now updating durable view for a potential primary _before_ it transitions to normal, so it _could_ be the case that `self.log_view == self.view and self.replica == self.primary_index(self.view) and self.status == .view_change` .